ios/swift/JitsiSDKTest/Podfile

```diff
@@ -1,5 +1,5 @@
 platform :ios, '12.0'
 
 target 'App' do
-    pod 'JitsiMeetSDK', '5.1.0'
+    pod 'JitsiMeetSDK', '5.2.0-alpha2'
```

Comment:
Do we need to depend on the alpha version for the example to work?

Generally speaking we use stable versions in the samples.

Reply:
the rotation is fixed only in the alpha version